utils/docker-compose.yaml

```diff
@@ -1,10 +1,24 @@
 version: '3'
 services:
+  orca-proxy:
+    build: ./orca_proxy
+    ports:
+      - "80:80"
+
   orca-ui:
-    build: ..
+    build:
+      context: ..
+      args:
+        - target=dev
     stdin_open: true
-    ports:
-      - "3000:3000"
+    network_mode: "service:orca-proxy"
```

Comment:
As far as I recognize, service discovery is enabled by default, and services are able to communicate using their names (`orca-proxy`, `orca-mock` etc.). Do we need the `network_mode` option for something else?

Reply:
Well, if I understand correctly, by using this network mode, this service is reachable from `orca-proxy` on `127.0.0.1:3000`.

But I think using service names is better so I removed `network_mode` and changed `nginx.conf` to use service names instead of `127.0.0.1`. Thanks for pointing that out

Reply:
I also modified `nginx.conf` so that the websockets are proxied properly - hot reload works correctly now.